package.json

```diff
@@ -303,6 +303,10 @@
             "type": "boolean",
             "default": true,
             "description": "Restart Tomcat Server or not when http(s) port changes"
+          },
+          "tomcat.auto_select_server": {
```

Comment:
I think the default value should be true.

Reply:
Its type is "string" because we want to support 'never' to let users have a choice not be always bothered with the option.